app/scripts/components/datasets/s-explore/timeseries/context.tsx

```diff
@@ -0,0 +1,35 @@
+import { ScaleTime, ZoomBehavior } from 'd3';
```

Comment:
It seems all the components that use this context are children componetns of `timeseries/index.tsx`. Why did you decided to use context here?

Reply:
When I stared I had a more nested structure which I was able to simplify. I decided to leave the context for scalability purposes. Would you say this complicated things and should be removed?

Reply:
I think using props generally shows how the data flows better. but maybe this will get used in somewhere outside of dateslider component in analysis page? I will leave this up to you.